src/content/containers/Home.js

```diff
@@ -33,6 +33,27 @@ type InstallButtonProps = {
   className?: string,
 };
 
+type UploadButtonProps = {
+  retrieveProfileFromFile: File => void,
+};
+
+class UploadButton extends PureComponent {
+  props: UploadButtonProps;
+
+  render() {
+    return (
+      <div>
+        <input type="file" ref="file"></input>
```

Comment:
nit: string refs are now deprecated in React (see https://facebook.github.io/react/docs/refs-and-the-dom.html#legacy-api-string-refs). Would you please change this to an inline function instead ?